azure-pipelines/nightly-snapshot-releases/Nightly Common VSTS Release.yml

```diff
@@ -1,15 +1,14 @@
-# 'Allow scripts to access the OAuth token' was selected in pipeline.  Add the following YAML to any steps requiring access:
-#       env:
-#           MY_ACCESS_TOKEN: $(System.AccessToken)
-# Variable 'android:serverUrl' was defined in the Variables tab
-# Variable 'ENV_VSTS_MVN_ANDROIDCOMMON_USERNAME' was defined in the Variables tab
-# Variable 'test_repo_branch' was defined in the Variables tab
-# Variable 'test_repo_dir' was defined in the Variables tab
-# Variable Group 'Android Common Variables' was defined in the Variables tab
-# Cron Schedules have been converted using UTC Time Zone and may need to be updated for your location
+# File: azure-pipelines\nightly-snapshot-releases\Nightly Common VSTS Release.yml
+# Description: Assemble & publish Snapshot common release nightly to internal feed
+name: $(date:yyyyMMdd)$(rev:.r)
```

Comment:
>$(date:yyyyMMdd)$(rev:.r) [](http://example.com/codeflow?start=6&length=25)

same here, if we can also add the version number here, that will be good.

Reply:
answered in previous comment